lib/wasix/src/lib.rs

```diff
@@ -170,6 +169,20 @@ pub enum SpawnError {
     /// Some other unhandled error. If you see this, it's probably a bug.
     #[error("unknown error found")]
     UnknownError,
+    #[error("runtime error")]
+    Runtime(#[from] WasiRuntimeError),
+    #[error("{0}")]
```

Comment:
Let's use `#[error(transparent)]` here so we don't print out the same error message twice.

```suggestion
    #[error(transparent)]
```